### PR TITLE
Updated putting data from armory into google sheets.

### PR DESCRIPTION
**Pull request by LCodepy**

### CODE REVIEW

File: services/rule_engine/src/main/java/agents/Moli.java [hunk 1]
```diff
@@ -19,11 +19,16 @@
...
 import com.mindsmiths.emailAdapter.EmailAdapterAPI;
+import com.mindsmiths.gsheetsAdapter.GSheetsAdapterAPI;
+import com.mindsmiths.gsheetsAdapter.reply.Spreadsheet;
 
 @Data
 @ToString(callSuper = true)
 @NoArgsConstructor
 public class Moli extends Agent {
+
+    String email = "test@gmail.com";
```
Comment (mjuraj) [inline services/rule_engine/src/main/java/agents/Moli.java:30]: You don't need a variable `email`. It is stored as a connection, and you can get it with `getConnection("email")`

File: services/rule_engine/src/main/java/agents/Smith.java [hunk 2]
```diff
@@ -10,18 +10,30 @@
...
+import com.mindsmiths.gsheetsAdapter.GSheetsAdapterAPI;
+import com.mindsmiths.gsheetsAdapter.reply.Spreadsheet;
+
 @Data
 @ToString(callSuper = true)
 @AllArgsConstructor
 public class Smith extends Agent {
 
     LocalDateTime lastTableUpdate = null;
+    int sheetSize = 0;
```
Comment (mjuraj) [inline services/rule_engine/src/main/java/agents/Smith.java:23]: This variable exists in the `prototype` branch

File: services/rule_engine/src/main/resources/rules/smith/Heartbeat.drl [hunk 3]
```diff
@@ -26,6 +26,15 @@ rule "Process Sheet"
         spreadsheet: Spreadsheet() from entry-point "signals"
         agent: Smith()
     then
-        agent.processSheet(spreadsheet);
+        modify(agent) {processSheet(spreadsheet)};
         delete(spreadsheet);
 end
+
+rule "Add Review"
```
Comment (mjuraj) [inline services/rule_engine/src/main/resources/rules/smith/Heartbeat.drl:33]: Call this rule something more descriptive. E.g. "Store review to Spreadsheet"

**Comment by NSocec-Mindsmiths** [comment]
I resolved the merge conflict, but I had to manually mix code from this branch and the target branch, so those parts should be reviewed again.

**Pull request merged**